moveit_simple/test/moveit_simple.cpp

```diff
@@ -289,10 +306,10 @@ TEST(MoveitSimpleTest, interpolation)
   for (std::size_t i = 0; i < joint_expected.size(); ++i)
   {
     error_joint += fabs(joint_out[i] - joint_expected[i]);
-    ROS_INFO_STREAM(" joint_out_ " << i << joint_out[i]);
-    ROS_INFO_STREAM(" joint_expected_ " << i << joint_expected[i]);
   }
   EXPECT_NEAR(error_joint, 0.0, 1e-2);
+  ROS_INFO_STREAM("joint_out" << vector_to_string(joint_out));
```

Comment:
Have you tried using just the stream operator, `<<`?  The prettyprint header should automatically make this work.